modules/op-creating-pipeline-run-pac.adoc

```diff
@@ -0,0 +1,157 @@
+// This module is included in the following assemblies:
+// * pac/creating-pipeline-runs-pac.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-pipeline-run-pac_{context}"]
+= Creating a pipeline run in {pac}
+
+Complete the following general steps to create a pipeline run definition for {pac} in your repository. Refer to subsequent sections for detailed reference information.
+
+.Prerequisites
+
+* You configured {pac} to integrate with your Git repository  hosting service provider.
+
+* You created a `Repository` custom resource (CR) to define the connection to your repository in {pac}.
+
+.Procedure
+
+. If a `.tekton` directory does not exist in the root of your repository, create this directory.
+
+. Create a file with a `.yaml` or `.yml` extension in the `.tekton` directory. In this file, create a YAML specification for a `PipelineRun` CR. This specification can use all features of {pipelines-shortname}.
+
+. Optional: Use parameters in your pipeline run specification. The {pac} resolver replaces these parameters with values that represent the current context. For example, `{{ repo_url }}` is the current URL for the repository and `{{ revision }}` is the branch on which the pipeline run was started. See the "Parameters in a pipeline run specification" section for a full list of the parameters.
```

Comment:
> See the "Parameters in a pipeline run specification" section for a full list of the parameters.

When you say  `See the Parameters in a pipeline run specification` how about giving a link to redirect :thinking: